src/test/java/tests/ShoppingCartTest.java

```diff
@@ -68,9 +68,8 @@ public void checkShoppingCart() throws InterruptedException {
 
         shoppingCartPage
                 .getCartProductTittle("2")
-                .clickValueBtn(1)
                 .selectValue("2");
-        sleep(5000);
+        sleep(3000);
```

Comment:
why do you need sleep here?

Reply:
Doesn't have time to change the price on the site when the selected product is increased... without sleep test failed.I tried to take the sleep to another place(on the "selectValue"), but the test passed every other time.Therefore, I reduced the duration of sleep to the minimum at which the test passes...

Reply:
Can you use some kind of waiter instead?

Reply:
I use waiter(on the "selectValue") but it's not enough